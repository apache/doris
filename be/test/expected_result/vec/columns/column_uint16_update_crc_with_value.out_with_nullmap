column: UInt16 with hashes: 2846380562 with ptr: 74