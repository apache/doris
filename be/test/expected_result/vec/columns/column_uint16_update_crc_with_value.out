column: UInt16 with hashes: 2752326018 with ptr: 74